.travis.yml

```diff
@@ -155,28 +157,34 @@ before_install:
     - echo -e "SHMEMCC=oshcc -std=c99\nSHMEMTOP=$$TRAVIS_INSTALL\n" > PRK/common/make.defs
     ## Build Libevent
     - cd $TRAVIS_SRC
-    - wget https://github.com/libevent/libevent/releases/download/release-2.0.22-stable/libevent-2.0.22-stable.tar.gz
-    - tar -xzvf libevent-2.0.22-stable.tar.gz
-    - cd libevent-2.0.22-stable
+    #- wget https://github.com/libevent/libevent/releases/download/release-2.0.22-stable/libevent-2.0.22-stable.tar.gz
+    - wget https://github.com/libevent/libevent/releases/download/release-2.1.8-stable/libevent-2.1.8-stable.tar.gz
```

Comment:
I had issues with this version and then I switched back to 2.1.8. There was a security failure. I am guessing you are able to work with this version. In that case, I will check again.

Reply:
Seems to be working.  Actually, I think I managed to also work around the PMIx failure.  Stay tuned ...